macros/edr/materializations/model/metrics.sql

```diff
@@ -0,0 +1,35 @@
+{% macro query_table_metrics() %}
+  {% set query %}
+    select
+      null as build,
```

Comment:
what is `null as build`?

Reply:
I wanted to create a metric to record the time of the model build.
This will be used for calculating the freshness.
Do you have a clearer metric name?
Maybe `build_time`?